### PR TITLE
mm : Modify about using mm_get_heap for NULL case

### PR DESCRIPTION
**Pull request by jeongchanKim**
1.     fs/procfs : Modify for getting Idle task's heap information in proc_entry_stat
    
    Idle task's stack pointer is not in the heap. So that mm_get_heap with idle task's stack ptr returns NULL.
    For getting idle task's heap information, called mm_get_heap_with_index(0).

2.     mm : Add NULL checking after mm_get_heap
    
    mm_get_heap can return NULL, but there are no error checking, so it can be crashed.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [4565b8aba1ced8e4a0eb1e68361cb85b7d0e094d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4565b8aba1ced8e4a0eb1e68361cb85b7d0e094d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [d22b69f15e6d0c416dbbd53e8d14575296f6ca00] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [d22b69f15e6d0c416dbbd53e8d14575296f6ca00] - Code Rule Check (C++) OK.

File: os/mm/umm_heap/umm_sem.c [hunk 1]
```diff
@@ -99,7 +100,11 @@ int umm_trysemaphore(void *address)
 {
 	struct mm_heap_s *heap;
 	heap = mm_get_heap(address);
-	return mm_trysemaphore(heap);
+	if (heap) {
+		return mm_trysemaphore(heap);
+	}
+	set_errno(EAGAIN);
+	return EAGAIN;
```
Comment (sunghan-chang) [inline os/mm/umm_heap/umm_sem.c]: Let's use another errno.

**Comment by seinfra** [comment]
Target : [31a7ef97a540199e9a276e1545598f6bb5462895] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [31a7ef97a540199e9a276e1545598f6bb5462895] - Code Rule Check (C++) OK.

**Pull request merged**